jib-maven-plugin/build.gradle

```diff
@@ -31,7 +31,7 @@ dependencies {
   testImplementation "com.github.stefanbirkner:system-rules:${dependencyVersions.SYSTEM_RULES}"
 
   testImplementation 'org.apache.maven.shared:maven-verifier:1.6'
-  testImplementation 'org.apache.maven:maven-compat:3.5.4'
```

Comment:
So we're downgrading this? Should everything be 3.5.4 instead?

Reply:
This downgrades the testing dependency; the end user doesn't see any changes. And although we can upgrade, at least it's safer not to upgrade the API. For example, we can guarantee users using Maven 3.5.2 will continue to work with Jib (although I think it will definitely work even if we upgrade this to 3.5.4.) The problem occurs when we use an API only available in 3.5.4 and the user uses 3.5.2, I think.